app/models/calculator/weight.rb

```diff
@@ -54,11 +54,14 @@ def weight_per_final_weight_volume(line_item)
     #    Customer ends up getting 350mL (line_item.final_weight_volume) of wine
     #      that represent 2.8 (quantity_implied_in_final_weight_volume) glasses of wine
     def quantity_implied_in_final_weight_volume(line_item)
+      return weight_per_variant(line_item) if line_item.variant.unit_value.zero?
+
       (1.0 * line_item.final_weight_volume / line_item.variant.unit_value).round(3)
     end
 
     def final_weight_volume_present?(line_item)
```

Comment:
This check was always returning `true`; the default value is a `BigDecimal` object with value `0.0`...

Reply:
I added it to fix a bug
e9e6aa77d88ebc6d93277d85bbc2e4eca1f558d2